3p/github.js

```diff
@@ -26,11 +26,11 @@ import {writeScript} from './3p';
  *
  * @param {!Window} global
  * @param {string} scriptSource The source of the script, different for post and comment embeds.
- * @param {function(*)} cb
+ * @param {function()} cb
  */
 function getGistJs(global, scriptSource, cb) {
   writeScript(global, scriptSource, function() {
-    cb(global.gist);
+    cb();
```

Comment:
Are we sure the callback will work without the argument passed? This seems like a change that would have an impact on the `cb` method.

Reply:
I double checked the callbacks for `gist` and `embedly` changes, params were unused

Reply:
Perfect! Will mark this as resolved.

Reply:
Yes, the alert wouldn't have fired if the params were used. The call site can is further down in the same file.

https://github.com/ampproject/amphtml/blob/c4a663d0ba13d0488c6fe73c55dc8c971ac6ec0d/3p/github.js#L55